www/js/controllers.js

```diff
@@ -8,18 +8,20 @@ angular.module('emission.controllers', ['emission.splash.updatecheck',
                                         'emission.survey.launch',
                                         'emission.stats.clientstats',
                                         'emission.incident.posttrip.prompt',
-                                        'customURLScheme'])
+                                        'customURLScheme',
+                                        'emission.splash.ionicNotification'])
```

Comment:
Don't work on the notifications yet. Let us get the current screen actually released first.